### ISSUE TITLE
Multiple conda error ending with "An unexpected error has occurred. Conda has prepared the above report."

### ISSUE THREAD

**Issue by Chandrima-04** [conda/conda#13167]
### Checklist

- [X] I added a descriptive title
- [X] I searched open reports and couldn't find a duplicate

### Summary

All functionality of conda has been failing.
I have tried creating a new environment, installing package, updating conda - this is the error:

```
(base) [chb4004@curie ~]$ which conda
~/miniconda3/bin/conda
(base) [chb4004@curie ~]$ conda --version
conda 23.5.0
(base) [chb4004@curie ~]$ conda update conda
Collecting package metadata (current_repodata.json): failed

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/gateways/repodata/__init__.py", line 187, in conda_http_errors
        yield
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/gateways/repodata/__init__.py", line 153, in repodata
        response.raise_for_status()
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
        raise HTTPError(http_error_msg, response=self)
    requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://conda.anaconda.org/biobakery/linux-64/current_repodata.json
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/requests/models.py", line 971, in json
        return complexjson.loads(self.text, **kwargs)
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/simplejson/__init__.py", line 525, in loads
        return _default_decoder.decode(s)
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/simplejson/decoder.py", line 370, in decode
        obj, end = self.raw_decode(s)
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/simplejson/decoder.py", line 400, in raw_decode
        return self.scan_once(s, idx=_w(s, idx).end())
    simplejson.errors.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/exception_handler.py", line 16, in __call__
        return func(*args, **kwargs)
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/cli/main.py", line 84, in main_subshell
        exit_code = do_call(args, p)
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/cli/conda_argparse.py", line 126, in do_call
        return getattr(module, func_name)(args, parser)
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/notices/core.py", line 123, in wrapper
        return func(*args, **kwargs)
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/cli/main_update.py", line 22, in execute
        install(args, parser, "update")
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/cli/install.py", line 309, in install
        unlink_link_transaction = solver.solve_for_transaction(
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/core/solve.py", line 153, in solve_for_transaction
        unlink_precs, link_precs = self.solve_for_diff(
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/core/solve.py", line 214, in solve_for_diff
        final_precs = self.solve_final_state(
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/core/solve.py", line 357, in solve_final_state
        ssc = self._collect_all_metadata(ssc)
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/common/io.py", line 83, in decorated
        return f(*args, **kwds)
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/core/solve.py", line 571, in _collect_all_metadata
        index, r = self._prepare(prepared_specs)
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/core/solve.py", line 1285, in _prepare
        reduced_index = get_reduced_index(
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/core/index.py", line 286, in get_reduced_index
        new_records = SubdirData.query_all(
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/core/subdir_data.py", line 157, in query_all
        result = tuple(
      File "/home/chb4004/miniconda3/lib/python3.9/concurrent/futures/_base.py", line 609, in result_iterator
        yield fs.pop().result()
      File "/home/chb4004/miniconda3/lib/python3.9/concurrent/futures/_base.py", line 439, in result
        return self.__get_result()
      File "/home/chb4004/miniconda3/lib/python3.9/concurrent/futures/_base.py", line 391, in __get_result
        raise self._exception
      File "/home/chb4004/miniconda3/lib/python3.9/concurrent/futures/thread.py", line 58, in run
        result = self.fn(*self.args, **self.kwargs)
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/core/subdir_data.py", line 142, in subdir_query
        return tuple(
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/core/subdir_data.py", line 164, in query
        self.load()
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/core/subdir_data.py", line 264, in load
        _internal_state = self._load()
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/core/subdir_data.py", line 321, in _load
        repodata, state = fetcher.fetch_latest_parsed()
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/gateways/repodata/__init__.py", line 757, in fetch_latest_parsed
        parsed, state = self.fetch_latest()
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/gateways/repodata/__init__.py", line 879, in fetch_latest
        raw_repodata = repo.repodata(cache.state)  # type: ignore
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/gateways/repodata/__init__.py", line 153, in repodata
        response.raise_for_status()
      File "/home/chb4004/miniconda3/lib/python3.9/contextlib.py", line 137, in __exit__
        self.gen.throw(typ, value, traceback)
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/gateways/repodata/__init__.py", line 234, in conda_http_errors
        raise RepodataIsEmpty(
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/conda/exceptions.py", line 527, in __init__
        body = response.json()
      File "/home/chb4004/miniconda3/lib/python3.9/site-packages/requests/models.py", line 975, in json
        raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
    requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

`$ /home/chb4004/miniconda3/bin/conda update conda`

  environment variables:
                 CIO_TEST=<not set>
        CONDA_DEFAULT_ENV=base
                CONDA_EXE=/home/chb4004/miniconda3/bin/conda
             CONDA_PREFIX=/home/chb4004/miniconda3
    CONDA_PROMPT_MODIFIER=(base)
         CONDA_PYTHON_EXE=/home/chb4004/miniconda3/bin/python
               CONDA_ROOT=/home/chb4004/miniconda3
              CONDA_SHLVL=1
           CURL_CA_BUNDLE=<not set>
               LD_PRELOAD=<not set>
               MODULEPATH=/pbtech_mounts/softlib001/apps/EL7/spack/share/spack/modules/linux-
                          centos7-x86_64:/usr/share/Modules/modulefiles:/etc/modulefiles
                     PATH=/home/chb4004/.cargo/bin:/home/chb4004/bin:/home/chb4004/miniconda3/bi
                          n:/home/chb4004/miniconda3/condabin:/pbtech_mounts/softlib001/apps/EL7
                          /spack/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/opt/ibut
                          ils/bin:/home/chb4004/.local/bin:/home/chb4004/miniconda3/bin:/home/ch
                          b4004/bin
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : base
    active env location : /home/chb4004/miniconda3
            shell level : 1
       user config file : /home/chb4004/.condarc
 populated config files : /home/chb4004/.condarc
          conda version : 23.5.0
    conda-build version : not installed
         python version : 3.9.16.final.0
       virtual packages : __archspec=1=x86_64
                          __glibc=2.17=0
                          __linux=3.10.0=0
                          __unix=0=0
       base environment : /home/chb4004/miniconda3  (writable)
      conda av data dir : /home/chb4004/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/bioconda/linux-64
                          https://conda.anaconda.org/bioconda/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://conda.anaconda.org/biobakery/linux-64
                          https://conda.anaconda.org/biobakery/noarch
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /home/chb4004/miniconda3/pkgs
                          /home/chb4004/.conda/pkgs
       envs directories : /home/chb4004/miniconda3/envs
                          /home/chb4004/.conda/envs
               platform : linux-64
             user-agent : conda/23.5.0 requests/2.31.0 CPython/3.9.16 Linux/3.10.0-1062.9.1.el7.x86_64 centos/7.7.1908 glibc/2.17
                UID:GID : 9221:1027
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.

If submitted, this report will be used by core maintainers to improve
future releases of conda.
Would you like conda to send this report to the core maintainers? [y/N]: y
Upload successful.

Thank you for helping to improve conda.
Opt-in to always sending reports (and not see this message again)
by running

    $ conda config --set report_errors true
```

### Linked Issues & PRs

- [ ] https://github.com/conda/conda/issues/7695

**Comment by travishathaway** [comment]
Hi @Chandrima-04,

Thanks for filing this issue. Please see my comment in a similar issue to find a work around for your situation:

https://github.com/conda/conda/issues/12570#issuecomment-1497329803

TLDR: you will need to remove that "biobakery" channel from your channel configuration to solve this problem.

This is a known bug, and the maintainers are currently working on a fix. We will resolve this issue (or the one I linked to) when this problem has been resolved.

**Comment by dholth** [comment]
I see you are using conda 23.5.0 and simplejson. This fix merged after 23.5.0 may be related https://github.com/conda/conda/pull/12683
